docs/boards/nrf52840_dongle.md

```diff
@@ -59,14 +59,50 @@ Follow these steps:
 
     ![Nordic dongle retainer clip](../img/dongle_clip.jpg)
 
-1.  Depending on the programmer you're using, you may have to adapt the next
-    command line. Run our script for compiling/flashing Tock OS on your device:
+#### JLink
 
-    ```shell
-    $ ./deploy.py --board=nrf52840_dongle --programmer=jlink
+Depending on the programmer you're using, you may have to adapt the next
```

Comment:
Now that there are dedicated sections for each programmer, this first sentence is not required anymore.